docs/en/setup/backend/dynamic-config.md

```diff
@@ -64,8 +64,24 @@ configuration:
     clusterName: "default"
 ```
 
+
+## Dynamic Configuration Zookeeper Implementation
+
+[Zookeeper](https://github.com/apache/zookeeper) is also supported in DCS, to use it, please configure as follows:
```

Comment:
Still `DCS` be placed in the document.

Reply:
Ok

Reply:
Please change this.

Reply:
@zhaoyuguang refer to this: https://github.com/apache/skywalking/pull/2826#discussion_r294081484

Reply:
OK